src/compas/files/off.py

```diff
@@ -128,11 +96,11 @@ def read(self):
 
         """
         if not self.content:
-            return
+            raise RuntimeError('Import failed')
 
         header = next(self.content)
-        if not header.lower() == 'off':
-            return
+        if header.lower() != 'off':
+            raise RuntimeError('Not a valid OFF file')
```

Comment:
just out of curiosity, why `RuntimeError`

Reply:
> just out of curiosity, why `RuntimeError`

I reread the builtin exception list and I see now that I doesn't really fit.

TypeError instead?

Reply:
`TypeError` or `ValueError`

Reply:
i think there are quite a few other occurrences...

Reply:
I know, I might have used the wrong one but at least I was consistently wrong ;).

Reply:
Just `Exception` if you're in doubt. TypeError is not the right one either, because it's not related to the type system.